bin/tns

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env node
 
 "use strict";
+
+require("source-map-support/register");
```

Comment:
I believe this will fail if CLI is packed, as we do not have dependency to `source-map-support`